packages/datadog-plugin-next/test/integration-test/client.spec.js

```diff
@@ -16,27 +17,27 @@ describe('esm', () => {
   let proc
   let sandbox
 
-  before(async function () {
-    // next builds slower in the CI, match timeout with unit tests
-    this.timeout(120 * 1000)
-    sandbox = await createSandbox(['next', 'react', 'react-dom'],
-      false, ['./packages/datadog-plugin-next/test/integration-test/*'], 'yarn exec next build')
-  })
+  withVersions('next', 'next', DD_MAJOR >= 4 && '>=11', version => {
```

Comment:
Why the explicit range here? And wouldn't this be false` for release lines _less_ than 4? Would that even work?